verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -1056,6 +1056,17 @@ paths:
                     $ref: '#/components/responses/422'
                 '500':
                     $ref: '#/components/responses/500'
+        delete:
+            tags: [User]
+            summary: Deletes a user
+            operationId: deleteUser
+            responses:
+                '204':
```

Comment:
I've found that if you don't explicitly list 401/403 here then the openapi validator middleware turns that error into a 500 (because you're doing something outside what's documented).